pwndbg/aglib/godbg.py

```diff
@@ -51,9 +53,15 @@ def word_size() -> int:
 
     Values taken from https://github.com/golang/go/blob/20b79fd5775c39061d949569743912ad5e58b0e7/src/go/types/sizes.go#L233-L252
     """
-    return {"i386": 4, "x86-64": 8, "aarch64": 8, "arm": 4, "rv64": 8, "powerpc": 8, "sparc": 8}[
-        pwndbg.aglib.arch.name
-    ]
+    return {
+        "i386": 4,
+        "x86-64": 8,
+        "aarch64": 8,
+        "arm": 4,
+        "rv64": 8,
+        "powerpc": 8,
+        "sparc": 8,
```

Comment:
probably s390x and loongarch64 are missing here, can you create another PR for adding it 👍

Reply:
It's a little awkward since this PR reformats the dict, but I can make another PR after this one gets merged. Should just be 8 for both architectures.